config/deploy/production.rb

```diff
@@ -1,9 +1,15 @@
-set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask(:deploy_host, '')
+set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask('deploy_host (leave off stanford.edu)', '')
 set :user, ENV['CAPISTRANO_PRODUCTION_USER'] || ask(:user, '')
+set :bundle_without, %w(deployment development test).join(' ')
```

Comment:
if so, that would be cool, but i think this just determines how bundler is called by capistrano when deploying - we use it in other projects i deploy with (and SW does too)

when he has a chance @cbeer can let us know